apps/scroll-cd/src/app/app.component.ts

```diff
@@ -33,14 +33,21 @@ import { BehaviorSubject } from 'rxjs';
 })
 export class AppComponent {
   title = 'scroll-cd';
-
+  ngZone: NgZone = inject(NgZone);
   private displayButtonSubject = new BehaviorSubject<boolean>(false);
   displayButton$ = this.displayButtonSubject.asObservable();
 
+  ngOnInit() {
+    this.ngZone.runOutsideAngular(() => {
+      window.addEventListener('scroll', this.onScroll.bind(this));
+    });
+  }
```

Comment:
This is not working. How did you test it ? Cd is still triggered at each CD cycle